### ISSUE TITLE
A bit laggy

### ISSUE THREAD

**Issue by Superchet** [thisgamesux/linux-csgo-external#12]
So, I've tried it.
It works pretty well, although I the glow ESP for the enemy either stops when it's between two walls or I'm not looking almost directly at them, my team ESP is fine, though.

**Comment by olaulau** [comment]
yes, it's seems that matchmaking competitive servers are a little more strict than standards servers.
so sometimes, glow are not on enemies if they are too far / if they are many walls between you.
so at round restart, enemies are far, you often see their glow on the map. it depends on which position your client received last from the server.
all of that doesn't happen on other types of servers, which seems to always send all players positions.

**Comment by Superchet** [comment]
This happens on everything except for Arms Race, though

**Comment by Oper8or** [comment]
I found that my client was able to see fairly far across the map and at all angles. I am not sure why it did though.

**Comment by Superchet** [comment]
Oper8or did you mess around with the code?

**Comment by Oper8or** [comment]
just minor stuff like colours. i did have high settings and custom tickrate

**Comment by Superchet** [comment]
Maybe weird to ask, but can you send me your folder and what custom tickrate you have?

**Comment by Oper8or** [comment]
cant send the code but the tickrate is 128

**Comment by Superchet** [comment]
Thanks
CSGO settings?

**Comment by Oper8or** [comment]
everything high

**Comment by Oper8or** [comment]
Did it help at all?

**Comment by Superchet** [comment]
Late reply, but not really, I think this is my side. Thanks, though!
